benchmark/largesetresult/largesetresult_test.go

```diff
@@ -28,6 +29,39 @@ func BenchmarkLargeResultSet(*testing.B) {
 	runLargeResultSet()
 }
 
+// getDSN constructs a DSN based on the test connection parameters
+func getDSN() (string, *sf.Config, error) {
+	env := func(k string, check bool) string {
+		if value := os.Getenv(k); value != "" {
+			return value
+		}
+		if check {
+			log.Fatalf("%v environment variable is not set.", k)
+		}
+		return ""
+	}
+
+	account := env("SNOWFLAKE_TEST_ACCOUNT", true)
+	user := env("SNOWFLAKE_TEST_USER", true)
+	password := env("SNOWFLAKE_TEST_PASSWORD", true)
+	host := env("SNOWFLAKE_TEST_HOST", false)
+	port := env("SNOWFLAKE_TEST_PORT", false)
+	protocol := env("SNOWFLAKE_TEST_PROTOCOL", false)
+
+	portStr, _ := strconv.Atoi(port)
+	cfg := &sf.Config{
```

Comment:
All `cmd` example codes but oauth uses `sf.Config` and `sf.DSN` to construct a DSN.